docs/application/toc_all.md

```diff
@@ -215,11 +215,11 @@
 ### User interface
 #### NUI
 ##### [Overview](/application/dotnet/guides/user-interface/nui/overview.md)
-
+##### [Layer](/application/dotnet/guides/user-interface/nui/layer.md)
+##### [Window](/application/dotnet/guides/user-interface/nui/window.md)
```

Comment:
Question to these lines.

In this PR, guides/user-interface/nui/layer.md and guides/user-interface/nui/window.md are not touched. Why are you doubling the menu for `Layer` and `Window`?
When you check on stg, you can see there are two `Layer` and `Window` in LNB.

Reply:
Fixed